# Params for CVX planner
#debug: true

use_ff: true # use feedforward

wdx: 20.0  #[m] world dimension in x
wdy: 20.0  #[m] world dimension in y
wdz: 4.0   #[m] world dimension in z
res: 0.15  #[m] cell dimension


dc: 0.01            #(seconds) Duration for the interpolation=Value of the timer pubGoal
goal_radius: 0.2    #(m) Drone has arrived to the goal when distance_to_goal<GOAL_RADIUS
drone_radius: 0.15  #(m) Used for collision checking


offset: 5  # Replanning offset (the initial conditions are taken OFFSET states farther from the last published goal)

Ra: 2.0             # [m] Radius of the first sphere
Rb: 6.0             # [m] Radius of the second sphere
w_max: 1            # [rd/s] Maximum angular velocity
alpha_0: 1          #[rd] threshold to ignore current JPS solution, and consider the old one
z_ground: 0         #[m] points below this are considered ground
inflation_jps: 0.8  #[m] The obstacles are inflated (to run JPS) by this amount


v_max: 2.0   #[m/s]
a_max: 2.0   #[m/s2]
j_max: 10.0  #[m/s3]


z_land: 0.02 #[m] z goal for landing



#Ways to make the planner faster:
# Increase deltaT in each cvxgen iteration
# Omit visualization stuff
# Timer of the mapper

#Things to try
#Change radius spheres Sa and Sb
#verify publish index and offset are working

